src/lumigo_tracer/utils.py

```diff
@@ -20,12 +20,14 @@
 MAX_VAR_LEN = 200
 MAX_ENTRY_SIZE = 1024
 FrameVariables = Dict[str, str]
-OMITTING_KEYS_REGEXES = [".*pass.*", ".*key.*"]
+OMITTING_KEYS_REGEXES = [".*pass.*", ".*key.*", ".*secret.*", ".*credential.*", ".*passphrase.*"]
```

Comment:
This is changing the default behavior of the tracer, it means that if someone updates the tracer the behavior will change - are we ok with that?

Reply:
yes,
I asked Aviad about it :)